src/main/java/pl/isa/freshmenindustries/Main.java

```diff
@@ -1,7 +1,74 @@
 package pl.isa.freshmenindustries;
+import java.util.InputMismatchException;
+import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
+        BasicOptions basicOptions = new BasicOptions();
+        basicOptions.displayOptions();
+
+        System.out.println(basicOptions.enterNumber());
+        try {
+            Scanner scanner = new Scanner(System.in);
+            int number = scanner.nextInt();
+            switch (number) {
+                case 1:
+                    System.out.println(basicOptions.chooseOption(number));
+                    System.out.println("You started a new game");
+                    break;
+                case 2:
+                    System.out.println(basicOptions.chooseOption(number));
+                    System.out.println("You are in manage games menu");
+                    break;
+                default:
+                    while (number != 1 || number != 2) {
```

Comment:
Jak widzisz pętla while jest w stanie Ci ciągle wczytywać dane od użytkownika, dopóki nie wprowadzi poprawnej wartości. To oznacza, że to ona powinna głównym elementem tej funkcji, a switch znajdować się wewnątrz niej. Jeżeli musisz coś napisać 2 razy w tym samym miejscu, to zapewne da się to napisać inaczej.

Reply:
Dodałam zmiany.